python/pyspark/sql/protobuf/functions.py

```diff
@@ -32,7 +32,7 @@
 def from_protobuf(
     data: "ColumnOrName",
     messageName: str,
-    descFilePath: str,
+    descFilePath: Optional[str] = None,
```

Comment:
Update documentation to reflect change in contract.
@HyukjinKwon is it common to use Optional[] for optional arguments in PySpark?

Reply:
Yes, it should be fine.